History.md

```diff
@@ -199,6 +199,10 @@
   and `html` files have been updated to use this new API. Read more on the
   [Wiki page](https://github.com/meteor/meteor/wiki/Build-Plugins-API).
 
+* There is a new API for adding assets to packages, `api.addAssets`. It works
```

Comment:
I think we shouldn't rely on people reading History.md to catch breaking changes.

There's several ways people will discover this change:
1. New documentation for `addAssets`
2. We print a nice error if you use the `isAsset` option
3. The error we print if you do addFiles on a file without a source handler also tells you to use `addAssets`
4. We will put this in the upgrade guide

It seems like this change is pretty hard to miss, and the errors printed should be pretty clear.